crates/runtime/Cargo.toml

```diff
@@ -14,6 +14,7 @@ owo-colors = { workspace = true, optional = true }
 rand.workspace = true
 serde = { workspace = true, features = ["derive"] }
 thiserror.workspace = true
+tokio = { workspace = true, features = ["fs", "io"] }
```

Comment:
this should be in `dev-dependencies`, optional by default and the `[[example]]` section should then have `required-features = ["tokio"]`